core-web/libs/portlets/edit-ema/portlet/src/lib/components/dot-block-editor-sidebar/dot-block-editor-sidebar.component.spec.ts

```diff
@@ -120,6 +115,9 @@ describe('DotBlockEditorSidebarComponent', () => {
         dotContentTypeService = spectator.inject(DotContentTypeService, true);
         dotAlertConfirmService = spectator.inject(DotAlertConfirmService, true);
         dotWorkflowActionsFireService = spectator.inject(DotWorkflowActionsFireService, true);
+
+        jest.spyOn(dotContentTypeService, 'getContentType').mockReturnValue(of(contentTypeMock));
```

Comment:
I suggest adding this part to the `mockProvider` definition, like this:

```ts
mockProvider(DotContentTypeService, {
    getContentType: jest.fn().mockReturnValue(of(contentTypeMock))
})
```

Reply:
tried that, not sure why gives errors... I will keep it as it is.